conda/recipes/rapids_core_dependencies/meta.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
```

Comment:
To verify my assumptions:
- We only release updates to the core dependencies on-cycle with the rest of RAPIDS releases (or hotfix)
- Updates to the dependencies come from `versions.json` and thus are part of the repo's commit hashes (new dependency versions = new commit hash = new package)

Reply:
Where does `VERSION_SUFFIX` come from? Our CI? It's not supplied by conda-build afaict.

Reply:
Those assumptions are correct ( excluding nightlies, but the git hash ensures uniqueness ).

The `VERSION_SUFFIX` should come from our CI ( it is listed in the `script_env` section). I based this on the RMM setup for Github Actions.  Looking at librmm nightlies I see them with the expected `VERSION_SUFFIX` of yymmdd`